docs/MAINTAINERS.adoc

```diff
@@ -0,0 +1,200 @@
+Remoting updates in the core are subject to the process though.
+
+== Team
+
+=== Communication channels
+
+* link:https://groups.google.com/d/forum/jenkinsci-dev[Jenkins Developer Mailing List]
+* Chat: N/A, to be created
+* Office hours: N/A, to be created
+
+=== Roles
+
+* Contributor
+* Core Pull Request Reviewer
+* Core Maintainer
+* Release Team Member
+
+**Contributors** submit pull requests to the Jenkins core and review changes submitted by others.
+They can also review and triage incoming issues in Jenkins Jira.
+There is no special preconditions to do so, anyone is welcome to contribute.
+
+**Core Pull Request Reviewers** is the next step for contributors who are interested to review Jenkins pull requests on a regular basis.
+They get [Triage permissions](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization) so that they can manage pull requests, request reviews and prepare changelog drafts in the pull request description.
+Their main responsibility is to triage and review the incoming pull requests,
+and to guide newcomer contributors who are not familiar with the project's processes.
+GitHub team: link:https://github.com/orgs/jenkinsci/teams/core-pr-reviewers[@jenkinsci/core-pr-reviewers].
+
+**Core Maintainers** get `Write` permissions in the repository, and hence they are able to merge pull requests.
+Their responsibility is to perform pull request reviews on a regular basis and to merge ready pull requests towards the weekly releases (`master` branch).
+They are also responsible to monitor the weekly release status and to perform triage of critical issues.
+GitHub team: link:https://github.com/orgs/jenkinsci/teams/core[@jenkinsci/core].
+
+**Release Team Members** are responsible for Jenkins weekly and LTS releases.
+Led by the link:https://jenkins.io/project/team-leads/#release[Jenkins Release Officer], they initiate releases, prepare changelogs and backport changes into the link:TODO[Stable branch].
+Team members get `Admin` permissions in the Jenkins core repository, and they also get permissions to trigger release Pipelines.
+
+=== Ladder
+
+* *Contributors*. Anyone can participate.
+  There is no precondition except having a GitHub account, just submit pull requests or comment in existing ones!
+* **Core Pull Request Reviewers** can be nominated by contributors in the link:https://groups.google.com/d/forum/jenkinsci-dev[developer mailing list]. 
+Self-nomination is fine.
+Decision is made by a consensus in the mailing list or via voting at the link:https://jenkins.io/project/governance-meeting/[governance meeting].
+* **Core Maintainers** - same process as for Core PR reviewers.
+  All nominees must also sign an link:https://github.com/jenkinsci/infra-cla/[Individual Contributor License Agreement] before getting the permission in GitHub repositories.
+* **Release Team Members** are assigned by the Jenkins Release officer
```

Comment:
Strictly speaking, the final boss is the release officer, elected every year? 😄

Reply:
Yup